### ISSUE TITLE
MP10 Source isn't actually uploaded

### ISSUE THREAD

**Issue by ChaseCorp** [Monoprice3DP/MP10Source#1]
Only the license and readme were posted, can you add the actual source please?

**Comment by trav0r** [comment]
So is this going to happen? I was really excited when I saw this repo show up then my heart was broken when I saw there wasn't actually anything here......

**Comment by ChaseCorp** [comment]
> So is this going to happen? I was really excited when I saw this repo show up then my heart was broken when I saw there wasn't actually anything here......

Hey actually I got ahold of monoprice and got a copy of the firmware. You can get it here https://github.com/ChaseCorp/Monoprice-MP10-source

**Comment by trav0r** [comment]
YES! I feel like I just woke up Christmas morning to find an awesome gift. Thank you sir!
